files/rules.d/mysql.yml

```diff
@@ -0,0 +1,28 @@
+# This file is managed by Puppet
+groups:
+  - name: mysql_Monitoring
```

Comment:
All group names should have a specific format, for example softwarename_rule or softwarename_whattheruledoes